src/main/java/com/rybka/config/ExchangeSource.java

```diff
@@ -1,13 +1,6 @@
 package com.rybka.config;
 
-import lombok.Getter;
-import lombok.RequiredArgsConstructor;
-
-@RequiredArgsConstructor
-@Getter
-public enum ExchangeSource {
```

Comment:
Why do you remove enum?

Reply:
enum value isn't constant for component name